.github/ISSUE_TEMPLATE/rendering-bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Rendering bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**LaTeX code**
+The code of LaTeX you tried to render:
+```latex
+
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen. You can obtain a LaTeX rendering using [QuickLaTeX](https://www.quicklatex.com/).
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
In this case, maybe we should encourage screenshots from the tester on `katex.org`, so that they double check it's not their environment?  And/or perhaps in Environment we should ask something like "- (if applicable) Custom web environment and KaTeX version"?